src/components/BasicHead.vue

```diff
@@ -189,10 +189,19 @@ export default {
         }
       }
     },
-    route$: {
+    '$route.name': {
       immediate: true,
       handler (val) {
         let name = this.$route.name
+	if(val) {
+	  name = val
+	}
+	if(name == 'oneview') {
```

Comment:
hard coded values cannot be used, nor can this value simple be provided as a prop - there may be multiple side views
instead, perhaps use the `view` structure as per https://github.com/voxgig/model-vue/blob/main/src/components/BasicSide.vue#L91 and check if the view is custom